kolibri/core/assets/src/vue/icon-button.vue

```diff
@@ -1,6 +1,14 @@
 <template>
 
-  <button @click="$emit('click')" class="icon-button-scope" :class="{'primary' : primary, 'single-line': !showTextBelowIcon}">
+  <button 
+    type="button" 
+    @click="$emit('click')"
+    @keypress.enter.prevent = "$emit('enter')"
```

Comment:
I can't say I completely understand the bug you experienced; however I tried to get 'enter' behaviors to work as I would expect here: 4184259f76488ab2c2d2299280e5bffc1bbdf1a3

Basically, I would expect:

* buttons trigger `click` events when they have keyboard focus and the enter key is pressed
* the form as a whole is also listening for enter keypresses, and should interpret those as submission of whatever the default action is
* when there is a conflict between the two above, the more specific action - the button with keyboard focus - takes precedence.